lib/govuk_navigation_helpers/related_navigation_sidebar.rb

```diff
@@ -39,6 +49,10 @@ def related_organisations
       build_links_for_sidebar(@content_item.related_organisations)
     end
 
+    def related_world_locations
```

Comment:
I'd be inclined to manipulate the collection to produce the correct path before passing it to `build_links_for_sidebar`.
This way you (ultimately) use the same data structuring method for all links.

Perhaps something like:
```ruby
def related_world_locations
  locations = @content_item.related_world_locations
  locations.map!{ |link| link.merge("base_path" => world_location_base_path(link["title"]))  }
  build_links_for_sidebar(locations)
end

private
def world_location_base_path(title)
  "/world/#{title.downcase.tr(' ', '-')}/news"
end
```